numero/numero.cpp

```diff
@@ -77,41 +77,38 @@ void load_time_zone(std::filesystem::path& exe) {
  *  Prepares for the computation by parsing the options and doing initial setup
  */
 std::ifstream prepare(int argc, char** argv) {
-	const char *OPT_HELP = "h", *OPT_OUTPUT_LINES = "n", *OPT_TIME_ZONE_ID = "tz", *OPT_INPUT_FILE = "input-file";
+	const std::string OPT_HELP{"help"}, OPT_OUTPUT_LINES{"output-lines"}, OPT_TIME_ZONE_ID{"time-zone"}, OPT_INPUT_FILE{"input-file"};
 	std::string input_file;
 
 	boost::program_options::options_description options{"Options"};
 	options.add_options()
-		(OPT_HELP, "produce help message")
-		(OPT_OUTPUT_LINES, boost::program_options::value<numero::options::output_lines_number_t>(&numero::options::output_lines_number)->default_value(100), "limit number of holidays in the output to n lines")
-		(OPT_TIME_ZONE_ID, boost::program_options::value<std::string>(&time_zone_string)->default_value(""), "specify output time zone (see README for details)")
+		((OPT_HELP + ",h").c_str(), "produce help message")
```

Comment:
nit: would be great to have short names defined somewhere nearby to long names (above).

Reply:
Done